doc/operations/certificates.rst

```diff
@@ -78,3 +78,49 @@ The procedure that operators and members should follow is summarised in the foll
+
+1. Get a globally reachable DNS name for your CCF network, e.g. ``my-ccf.example.com``, which resolves to the address of at least one node in the network. Multiple nodes or a load balancer address are fine too.
+
+2. ACME `http-01 <https://letsencrypt.org/docs/challenge-types/>`_ challenges require a challenge server to be reachable on port 80 (non-negotiable). 
+To be able to bind to that port, the ``cchost`` binary may need to be given special permission, e.g. by running ``sudo setcap CAP_NET_BIND_SERVICE=+eip cchost``. Alternatively, port 80 can be redirected to a non-privileged port that ``chost`` may bind to without special permission.
+
+3. Each interface defined in the ``chost`` configuration file can be given the name of an ACME configuration to use. The settings of each ACME configuration are defined in ``network.acme``. Note that this information is required by *all* nodes as they might have to renew the certificate(s) later.
+
+    The various options are as follows:
+
+    .. code-block:: json
+
+        "network": {
+            "rpc_interfaces": { 
+                ... ,
+                "acme_endorsed_interface": { ... , "acme_configuration": "my-acme-cfg" }
+            },
+            "acme": {
+                "my-acme-cfg": { 
+                    "ca_certs": [ "-----BEGIN CERTIFICATE-----\nMIIBg ..." ],
+                    "directory_url": "https://...",
+                    "service_dns_name": "my-ccf.example.com",        
+                    "contact": ["mailto:john@example.com"],
+                    "terms_of_service_agreed": true,
+                    "challenge_type": "http-01",
+                    "challenge_server_interface": "0.0.0.0:80"
+                }
+            }
+        }
+
+
+    - ``ca_certs``: CCF will need to establish https connections with the CA, but does not come with root certificates by default and therefore will fail to establish connections. This setting is populated with on or more such certificates; e.g. for Let's Encrypt this would be their ISRG Root X1 certificate (see `here <https://letsencrypt.org/certificates/>`_) in PEM format.
```

Comment:
```suggestion
    - ``ca_certs``: CCF will need to establish https connections with the CA, but does not come with root certificates by default and therefore will fail to establish connections. This setting is populated with one or more such certificates; e.g. for Let's Encrypt this would be their ISRG Root X1 certificate (see `here <https://letsencrypt.org/certificates/>`_) in PEM format.
```